src/Http/Controllers/Auth/PasswordResetController.php

```diff
@@ -44,16 +45,32 @@ public function store(PasswordResetRequest $request): View|RedirectResponse
         return back()->withErrors(["email" => __($status)]);
     }
 
-    public function edit(PasswordResetRequest $request, string $token): View
+    public function edit(string $token): View
     {
-        return view("user.password.reset-password")->with(["email" => $request->validated("email"), "token" => $token]);
+        $email = request()->get("email");
```

Comment:
Please inject Illuminate Request here and get this email from there.

Reply:
done